app/modules/auth/service.server.ts

```diff
@@ -299,6 +307,37 @@ export async function getAuthResponseByAccessToken(accessToken: string) {
   }
 }
 
+interface Session {
```

Comment:
What is this type? We already have a type for our session and it has more data inside it. If you want to use it with a Partial, just use the proper session type we have.

Reply:
ack

Reply:
> What is this type? We already have a type for our session and it has more data inside it. If you want to use it with a Partial, just use the proper session type we have.

I think it is just to type the query result but It can be removed. We just have to now if the query returns an array with a given length. `unknow[]` should be enough

Reply:
> > What is this type? We already have a type for our session and it has more data inside it. If you want to use it with a Partial, just use the proper session type we have.
> 
> I think it is just to type the query result but It can be removed. We just have to now if the query returns an array with a given length. `unknow[]` should be enough
we can do this. since we are using it only here, we can remove that interface and define it like below
 const result = await db.$queryRaw<{id:String, revoked:boolean}[]>`

Reply:
Thats good. I just didn't want to have this type there with such a generic name like Session as it can get confusing.